R/rtools.R

```diff
@@ -127,11 +127,10 @@ renv_rtools_envvars_default <- function(root) {
 
   # add Rtools utilities to path
   bin <- normalizePath(file.path(root, "bin"), mustWork = FALSE)
-
   path <- paste(bin, Sys.getenv("PATH"), sep = .Platform$path.sep)
 
   # set BINPREF (note: trailing slash is required)
-  binpref <- file.path(renv_path_normalize(root), "mingw_$(WIN)/bin/")
+  binpref <- paste(renv_path_normalize(root), "mingw_$(WIN)/bin/", sep = "/")
```

Comment:
FWIW I'm pretty sure `file.path()` uses the same separator on every platform.

Reply:
`file.path()` drops trailing separators on Windows.

```
> file.path("a/b/")
[1] "a/b"
```

Reply:
Ooooooh that bastard!